docs/source/optional_plugins/index.rst

```diff
@@ -18,3 +18,4 @@ optional plugins:
    yaml_loader
    golang
    glib
+   Cit
```

Comment:
I don't think we should mix upper and lower cases, would you please use `cit` instead?

Reply:
Agreed (s/Cit/cit/)

Reply:
ok